WalkingChallenge/Info.plist

```diff
@@ -2,30 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>NSAppTransportSecurity</key>
```

Comment:
*sigh* I really dislike the PropertyList/CFDictionary serialization being based on `CFHash`.  What was the change made to the plist?  Perhaps you can edit it by hand instead to avoid the reordering noise?

Reply:
I'm not exactly sure how this change even happened, you seem to have better insight on this than I. If you could explain what is going on here I would appreciate it.

Reply:
When you change the property list through Xcode, it will de-serialise and re-serialise it.  However, that is non-deterministic, and the keys can get re-ordered since the dictionary is unordered.  This means that its hard to spot the change.